tests/framework/cli/test_cli_hooks.py

```diff
@@ -108,3 +120,9 @@ def test_kedro_cli_should_invoke_cli_hooks_from_plugin(
             f"Before command `{command}` run for project {fake_metadata}"
             in result.output
         )
+
+        # 'pipeline list' and 'info' aren't actually in the click structure and
+        # return exit code 2 ('invalid usage of some shell built-in command')
```

Comment:
Do you happen to know why these command aren't in the click structure? Should they be?

Reply:
See above